bridge-token-factory/src/lib.rs

```diff
@@ -343,32 +336,26 @@ impl BridgeTokenFactory {
 
         let Recipient { target, message } = parse_recipient(new_owner_id);
 
-        env::log(format!("Finish deposit. Target:{} Message:{:?}", target, message).as_bytes());
+        env::log_str(&format!(
+            "Finish deposit. Target:{} Message:{:?}",
+            target, message
+        ));
 
         match message {
-            Some(message) => ext_bridge_token::mint(
-                env::current_account_id(),
-                amount.into(),
-                &self.get_bridge_token_account_id(token.clone()),
-                env::attached_deposit() - required_deposit,
-                MINT_GAS,
-            )
-            .then(ext_bridge_token::ft_transfer_call(
-                target.try_into().unwrap(),
-                amount.into(),
-                None,
-                message,
-                &self.get_bridge_token_account_id(token),
-                1,
-                FT_TRANSFER_CALL_GAS,
-            )),
-            None => ext_bridge_token::mint(
-                target,
-                amount.into(),
-                &self.get_bridge_token_account_id(token),
-                env::attached_deposit() - required_deposit,
-                MINT_GAS,
-            ),
+            Some(message) => ext_bridge_token::ext(self.get_bridge_token_account_id(token.clone()))
+                .with_static_gas(MINT_GAS)
+                .with_attached_deposit(env::attached_deposit() - required_deposit)
+                .mint(env::current_account_id(), amount.into())
+                .then(
+                    ext_bridge_token::ext(self.get_bridge_token_account_id(token))
+                        .with_static_gas(FT_TRANSFER_CALL_GAS)
+                        .with_attached_deposit(1)
+                        .ft_transfer_call(target.try_into().unwrap(), amount.into(), None, message),
```

Comment:
`target` is already `AccountId` type
```suggestion
                        .ft_transfer_call(target.unwrap(), amount.into(), None, message),
```

Reply:
Fixed 8e2b316371f19907869b1bd8506f976beb82ce31